web/app/js/components/TopEventTable.jsx

```diff
@@ -1,25 +1,21 @@
 import BaseTable from './BaseTable.jsx';
 import React from 'react';
-import { srcDstColumn } from './util/TapUtils.jsx';
 import { withContext } from './util/AppContext.jsx';
+import { directionColumn, srcDstColumn } from './util/TapUtils.jsx';
 import { formatLatencySec, numericSort } from './util/Utils.js';
 
-const srcDstSorter = key => {
-  return (a, b) => (a[key].pod || a[key].str).localeCompare(b[key].pod || b[key].str);
-};
-
 const topColumns = ResourceLink => [
   {
-    title: "Source",
-    key: "source",
-    sorter: srcDstSorter("source"),
-    render: d => srcDstColumn(d.source, d.sourceLabels, ResourceLink)
+    title: " ",
```

Comment:
curious: why no title in the Top table, but `Direction` title in the Tap table. i kind of like how it looks without title in the Top table. perhaps if we removed the `ID` column from Tap, we could render more similarly?

Reply:
Mostly because of the `ID` column, yeah :) I think we can totally remove the ID column and have this be the first column!